public/js/angular/filters/parsable_date.js

```diff
@@ -1,6 +1,8 @@
 angular.module('parseDate', [])
 .filter('parsableDate', function() {
     return function(input) {
+        if(typeof(input)==='undefined')
```

Comment:
For future reference, we wrap all conditionals in brackets, I'll make the change in master.